frame/support/procedural/src/pallet/parse/storage.rs

```diff
@@ -510,18 +513,28 @@ fn process_unnamed_generics(
 		err
 	})?;
 
+	let use_default_hasher = |arg_pos| {
+		if let Some(arg) = retrieve_arg(arg_pos).ok() {
+			dev_mode && syn::parse2::<syn::Token![_]>(arg.to_token_stream()).is_ok()
+		} else {
+			false
```

Comment:
Why is this just false if the argument does not exist?  
I thought it has to exist and be `_`?